recipe/meta.yaml

```
@@ -10,16 +10,16 @@ source:
   sha256: a14c1c5143081701d4fe62283e09cc5ad5336dff3a152de64eb77ee3180e9792
 
 build:
-  number: 1
+  number: 2
   noarch: python
 
 requirements:
   host:
     - python >=3.8
   run:
     - python >=3.8
-    - dask-core ={{ version }},!={{ version }}a*
-    - distributed ={{ version }},!={{ version }}a*
+    - dask-core {{ version }}.*,!={{ version }}a.*
+    - distributed {{ version }}.*,!={{ version }}a.*
```

If the `,`s are indeed causing issues, we could break these out as separate lines. Listing the same requirement multiple times with different constraints still behaves like an `and`, but may avoid any parsing bug that may be coming up

```suggestion
    - dask-core ={{ version }}
    - dask-core !={{ version }}a*
    - distributed ={{ version }}
    - distributed !={{ version }}a*
```

---

Tried these constraints out and rebuilt the packages and they still seem to allow nightly packages to be installed 😕 will try the same constraints with `.*` instead to see if that works, even though the documentation seems to imply it wouldn't

---

Strangely enough, things do work as expected when I use

```suggestion
    - dask-core ={{ version }}
    - dask-core !={{ version }}a.*
    - distributed ={{ version }}
    - distributed !={{ version }}a.*
```